ProjFS.Mac/PrjFSKext/VnodeCache.cpp

```diff
@@ -211,9 +228,59 @@ void VnodeCache_InvalidateCache(PerfTracer* _Nonnull perfTracer)
     RWLock_ReleaseExclusive(s_entriesLock);
 }
 
+IOReturn VnodeCache_ExportHealthData(IOExternalMethodArguments* _Nonnull arguments)
+{
+    PrjFSVnodeCacheHealth healthData =
+    {
+        .cacheCapacity = s_entriesCapacity,
+        .cacheEntries = s_cacheStats.cacheEntries, // cacheEntries is reset to 0 when VnodeCache_InvalidateCache is called
+        .invalidateEntireCacheCount = atomic_exchange_explicit(
+            &s_cacheStats.healthStats[VnodeCacheHealthStat_InvalidateEntireCacheCount], 0ULL, memory_order_relaxed),
+        .totalCacheLookups = atomic_exchange_explicit(&s_cacheStats.healthStats[VnodeCacheHealthStat_TotalCacheLookups], 0ULL, memory_order_relaxed),
+        .totalLookupCollisions = atomic_exchange_explicit(&s_cacheStats.healthStats[VnodeCacheHealthStat_TotalLookupCollisions], 0ULL, memory_order_relaxed),
+        .totalFindRootForVnodeHits = atomic_exchange_explicit(&s_cacheStats.healthStats[VnodeCacheHealthStat_InvalidateEntireCacheCount], 0ULL, memory_order_relaxed),
```

Comment:
> Why are we exporting VnodeCacheHealthStat_InvalidateEntireCacheCount as totalFindRootForVnodeHits?

This was definitely a bug and fix is in https://github.com/Microsoft/VFSForGit/pull/1041/commits/16bca67d6ded272e68fcf13453cbbff2dc00c875

I came across this same issue yesterday while doing some final testing because hit count was always being reported as 0 which seemed very suspicious